ts/index.ts

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable @typescript-eslint/ban-ts-comment */
 // @ts-ignore
-import { utils } from 'libp2p-pubsub'
+import { utils } from 'libp2p-pubsub' // TODO: change
```

Comment:
I would like some help on this. I tried to change this line like I did for the others, but ts did not like it

Reply:
resolved